packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -68,24 +73,40 @@ export class Tooltip extends React.Component {
   }
 
   setTooltipActive(active) {
-    this.setState({ active }, () => {
-      this.popper.forceUpdate();
-    });
+    if (active !== this.state.active) {
+      this.setState({ active }, () => {
+        this.popper.forceUpdate();
+        if (active) {
+          this.props.onOpen && this.props.onOpen();
+        } else {
+          this.props.onClose && this.props.onClose();
+        }
+      });
+    }
   }
 
   handleBlur() {
     // Hide tooltips when blurring away from the trigger or tooltip body
+    // and when the mouse is not hovering over the tooltip
     setTimeout(() => {
       const focusedInsideTrigger =
         this.triggerElement && this.triggerElement.contains(document.activeElement);
       const focusedInsideTooltip =
         this.tooltipElement && this.tooltipElement.contains(document.activeElement);
-      if (!focusedInsideTrigger && !focusedInsideTooltip) {
+      if (!focusedInsideTrigger && !focusedInsideTooltip && !this.state.isHover) {
```

Comment:
If focus moves to another tooltip, but the mouse remains hovering over an opened tooltip, both tooltips will be open. 

This is consistent with the requirement that tooltips are activated on hover and focus